examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -79,7 +81,9 @@
         "availabilityProfile": "VirtualMachineScaleSets",
         "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
         "osDiskCachingType": "ReadOnly",
-        "dataDiskCachingType": "ReadWrite"
+        "dataDiskCachingType": "ReadWrite",
+        "osDiskSSDType": "Standard",
```

Comment:
instead of SSD type, have you considered calling this `osDiskStorageAccountType`? I would 1) be consistent with other usages across Azure and 2) allow for Standard_HDD and any other non-SSD types that may be added in the future

Reply:
How about just `osDiskType` and `dataDiskType`?

Just reading this doc, that would be an accurate concise way to represent this with a property name:

https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types

Reply:
Possible values:

- "StandardHDD"
- "StandardSSD"
- "PremiumSSD"
- "UltraSSD"

Reply:
SGTM

Reply:
Only caveat, you might want different types per Data disk so having one global `dataDiskType` might not be the most flexible

Reply:
I think that inflexibility is already built into the AKS Engine config interface, as the nominal data disk config interface is simply an array of disk sizes; so the assumption is that they are all otherwise (besides size) similarly configured.

Definitely a gotcha for capz, tho